postman/collections/denner-api.postman-v2.json

```diff
@@ -2190,6 +2160,184 @@
+								"description": ""
+							},
+							{
+								"key": "App-ID",
+								"value": "{{advertising_header_app_id}}",
+								"description": ""
+							},
+							{
+								"key": "App-Key",
+								"value": "{{advertising_header_app_key}}",
+								"description": ""
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"description": "List a single link (Link) for a print publication (Print-Werbemittel)"
+					},
+					"response": []
+				},
+				{
+					"name": "/print-publications/{publication_id}/links",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									"// Save created publication article's id in environment",
+									"var jsonData = JSON.parse(responseBody),",
+									"    id = (jsonData) ? jsonData.id : null;",
+									"    ",
+									"postman.setEnvironmentVariable(\"publication_article_id\", id);",
+									"",
+									"tests[\"Setup new publication_article_id in environment\"] = !!id;",
+									""
+								]
+							}
+						}
+					],
+					"request": {
+						"url": "{{advertising_url_base}}/print-publications/{{publication_id}}/links",
+						"method": "POST",
+						"header": [
+							{
+								"key": "Accept",
+								"value": "{{header_accept}}",
+								"description": ""
+							},
+							{
+								"key": "App-ID",
+								"value": "{{advertising_header_app_id}}",
+								"description": ""
+							},
+							{
+								"key": "App-Key",
+								"value": "{{advertising_header_app_key}}",
+								"description": ""
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"description": ""
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n\t\"id\": \"{{link_id}}\",\n\t\"name\": \"Hans\",\n\t\"valid_to\": \"2017-08-28\"\n}"
+						},
+						"description": "Add a new link to a print publication (Print-Werbemittel)"
+					},
+					"response": []
+				},
+				{
+					"name": "/print-publications/{publication_id}/links/{link_id}",
+					"request": {
+						"url": "{{advertising_url_base}}/print-publications/{{publication_id}}/links/{{link_id}}",
+						"method": "PATCH",
+						"header": [
+							{
+								"key": "Accept",
+								"value": "{{header_accept}}",
+								"description": ""
+							},
+							{
+								"key": "App-ID",
+								"value": "{{advertising_header_app_id}}",
+								"description": ""
+							},
+							{
+								"key": "App-Key",
+								"value": "{{advertising_header_app_key}}",
+								"description": ""
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"description": ""
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n\t\"name\": \"Hans 2\",\n\t\"valid_to\": \"2017-08-29\"\n}"
+						},
+						"description": "Update Links's details from a print publication by listing (Print-Werbemittel)"
```

Comment:
Write "Link's" lowercase.

For the other operation you don't mention that links only work for publications "by listing". I think we don't need to mention it here.